scripts/src/index.ts

```diff
@@ -0,0 +1,330 @@
+    // if you decide to swap, you can do it like this:
+
+    // the price might change in the meantime, so we should apply slippage
+    // slippage is a price change you are willing to accept
+    // for example, if current price is 1 and your slippage is 1%, the price limit should be set to 1.01
+    const allowedSlippage = toPercentage(1n, 3n) // 0.001 = 0.1%
+
+    const sqrtPriceLimit = calculateSqrtPriceAfterSlippage(
+      simulateResult.targetSqrtPrice,
+      allowedSlippage,
+      false
+    )
+
+    const swapTransactionId = await invariant.swap(
+      account,
+      poolKey,
+      true,
+      amount,
+      true,
+      sqrtPriceLimit
+    )
+    // print swap transaction id
+    console.log(swapTransactionId)
+  }
+  // query state
+  const pool: Pool = await invariant.getPool(poolKey)
+  const position: Position = await invariant.getPosition(account.address, 0n)
+  const lowerTick: Tick = await invariant.getTick(poolKey, position.lowerTickIndex)
+  const upperTick: Tick = await invariant.getTick(poolKey, position.upperTickIndex)
+
+  // check amount of tokens you are able to claim
+  const fees = await calculateFee(pool, position, lowerTick, upperTick)
+
+  // print amount of unclaimed x and y tokens
+  console.log(fees)
+
+  expect(fees).to.deep.equal([59999999999n, 0n])
+
+  // get balance of a specific token before claiming position fees and print it
+  const accountBalanceBeforeClaim = await token.getBalanceOf(account.address, poolKey.tokenX)
+  console.log(accountBalanceBeforeClaim)
+
+  // specify position id
+  const positionId = 0n
+  // claim fee
+  const claimFeeTransactionId = await invariant.claimFee(account, positionId)
+  // print transaction hash
+  console.log(claimFeeTransactionId)
+
+  // get balance of a specific token before claiming position fees and print it
+  const accountBalanceAfterClaim = await token.getBalanceOf(account.address, poolKey.tokenX)
+  console.log(accountBalanceAfterClaim)
+
+  expect(accountBalanceAfterClaim).to.equal(accountBalanceBeforeClaim + fees[0])
+
+  const receiver = await getSigner(1000n * ONE_ALPH)
+
+  const { owner, ...positionToTransfer } = await invariant.getPosition(account.address, 0n)
+
+  // Transfer position from account (signer) to receiver
+  await invariant.transferPosition(account, 0n, receiver.address)
+  const receiverPosition = await invariant.getPosition(receiver.address, 0n)
+  // the position will be the same, except for the owner field
+  expect(receiverPosition).to.deep.include(positionToTransfer)
+  console.log(receiverPosition)
+
+  // ### transfer the position back to the original account
+  await invariant.transferPosition(receiver, 0n, account.address)
+  // ###
+
+  // fetch user balances before removal
+  const accountToken0BalanceBeforeRemove = await token.getBalanceOf(account.address, poolKey.tokenX)
+  const accountToken1BalanceBeforeRemove = await token.getBalanceOf(account.address, TOKEN1_ID)
```

Comment:
It does not look correct when we are taking id from Poolkey and from deploy result. IMO we should always suggest using poolKey. Change it in docs as well please